src/main/webapp/views/common/navbar.html

```diff
@@ -18,7 +18,7 @@
         </li>
       </ul>
       <form id="log_submit_form" class="form-inline my-2 my-lg-0">
-        <a id="log-button" class="btn btn-sm btn-outline-secondary">Log In</a>
+        <a id="log-button" class="btn btn-sm btn-outline-secondary">Log in as Vendor</a>
```

Comment:
You have the same `id` in dropdown.html, won't there be problems with it?

Reply:
There's no problem. In `setDropdownInDOM()` I first delete all the child's inside the form using `$(logFormTag).empty();` and then insert the dropdown `$(logFormTag).load('common/dropdown.html')`